cmd/gsutil/main.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Base image expects gsutil binary in $PATH
+gsutil command uses `-m` flag(default) for faster download and uploads.
+This file executes cp sub-command like `cp source destination`
+*/
```

Comment:
i think it would be helpful for all of our `main.go`'s to have some info about how to use the resulting binaries, e.g. something like:

```
This tool is for _______

To use it, run ____
For help, run ______

For example:

______

```

Reply:
@bobcatfish : I have updated comments to include new style. Let me know if thats good.